UnityProject/Assets/Scripts/Core/GameGizmos/GameGizmomanager.cs

```diff
@@ -81,11 +81,11 @@ public static GameGizmoBox AddNewBoxStaticClient(GameObject TrackingFrom, Vector
 	}
 
 
-	public GameGizmoLine AddNewLine(GameObject TrackingFrom, Vector3 From,   GameObject TrackingTo, Vector3 To, Color color, float LineThickness)
+	public GameGizmoLine AddNewLine(GameObject TrackingFrom, Vector3 From,   GameObject TrackingTo, Vector3 To, Color color, float LineThickness, float time = -1)
```

Comment:
Might as well add it to the base of all gizmos

Reply:
done